plexapi/playqueue.py

```diff
@@ -58,19 +58,37 @@ def _loadData(self, data):
         self.size = utils.cast(int, data.attrib.get("size", 0))
         self.items = self.findItems(data)
 
+    def __len__(self):
+        return len(self.items)
+
+    def __iter__(self):
+        for item in self.items:
+            yield item
+
     def __contains__(self, media):
         """Returns True if the PlayQueue contains the provided media item."""
-        return any(x == media for x in self.items)
+        return any(x.playQueueItemID == media.playQueueItemID for x in self.items)
 
-    def has_queueItemID(self, playQueueItemID):
-        """Returns True if the PlayQueue contains an item with the provided playQueueItemID."""
-        return any(x.playQueueItemID == playQueueItemID for x in self.items)
+    def get_queue_item(self, item):
```

Comment:
Shouldn't this follow the camel casing? `getQueueItem`

Reply:
Yeah, I'm torn on this. It's not returning a queueItem object, but a regular media item that has been enriched with a `playQueueItemID` attribute for being part of a `PlayQueue`: https://github.com/pkkid/python-plexapi/blob/5b47a3be0c96749e58adbaa4ac7f57c1e14f1c2e/plexapi/base.py#L601

I'm open to whatever feels more logical or consistent.

Reply:
Went with your suggestion and made it camel case for consistency.